addon/services/flexberry-logging.js

```diff
@@ -39,21 +62,26 @@ export default Ember.Service.extend({
    */
   init() {
     this._super(...arguments);
-    this.set('flexberryStore', getOwner(this).lookup('service:store'));
-    this.set('serverLogEnabled', true);
+    this.set('_flexberryStore', getOwner(this).lookup('service:store'));
```

Comment:
We can get store like this:
  /**
    Current store to request records.

    @property _store
    @private
    @type DS.Store
  */
  _store: Ember.inject.service('store'),